ote_sdk/ote_sdk/usecases/exportable_code/prediction_to_annotation_converter.py

```diff
@@ -274,13 +274,17 @@ def __init__(self, label_schema: LabelSchemaEntity):
     def convert_to_annotation(
         self, predictions: float, metadata: Dict[str, Any]
     ) -> AnnotationSceneEntity:
-        pred_label = predictions >= 0.5
+        pred_label = metadata["pred_score"] >= 0.5
```

Comment:
Is this always the case, including min-max and cdf normalisations, maybe other implementations we might add in the future? We could always do the following without modifying this function:

```
predictions = metadata["pred"]
annotation = convert_to_annotation(predictions, metadata)
```

Am I missing anything here?

Reply:
We can remove the anomaly masks from the `postprocess` function or move that to metadata and return the scores as predictions. But we can't do anything in the between as it is called from SyncDemo 
![image](https://user-images.githubusercontent.com/17232914/148029043-2c2ed808-acd5-4e07-a1e0-96646182eaac.png)